listenbrainz_spark/utils.py

```diff
@@ -64,13 +63,15 @@ def get_listens(d1, d2):
                     'recording_msid', 'release_mbid', 'release_msid', 'release_name', 'tags',
                     'track_name', 'user_name'
                 ]
+        Note: Dataframe of current month would not be included.
```

Comment:
@mayhem The note here indicates that the current date/end date would not be included.